llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -977,9 +977,19 @@ def TuneLUIADDIFusion
 def TuneAUIPCADDIFusion
     : SubtargetFeature<"auipc-addi-fusion", "HasAUIPCADDIFusion",
                        "true", "Enable AUIPC+ADDI macrofusion">;
-def TuneShiftedZExtFusion
-    : SubtargetFeature<"shifted-zext-fusion", "HasShiftedZExtFusion",
-                       "true", "Enable SLLI+SRLI to be fused when computing (shifted) zero extension">;
+
```

Comment:
I don't think "word" is the right word here ) Machine word on riscv64 is 64 bits. How about `ZExt16` and `ZExt32` instead?

Reply:
Or even if "word" is the right terminology here, I'd still prefer numeric suffixes because they are more explicit

Reply:
zextw and zexth are consistent with the names of the instructions that were added in Zba and Zbb to replace these shift patterns. But I don't have a strong preference.

Reply:
I stand for names `zexth` and `zextw`,  as @topperc said, they are the names for equivalent existed instructions. I think people may be much more familiar with these. WDYT?

Reply:
Yes, sounds good to me. I didn't know about these.